src/Aspire.Hosting.Python/FlaskProjectResourceExtensionBuilder.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting.Python;
+
+/// <summary>
+/// Provides extension methods for adding Python Flask applications to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class FlaskProjectResourceExtensionBuilder
```

Comment:
I think this could probably collapsed into a single extension class with all the python-related extension methods in it.

Reply:
For the Flask extension that's certainly the case. I'm also working on Django right now, and that's a little more complicated. For Django to run well, you'll need to have two things:

- A super user which you can create with a specific python command
- Applied migrations which requires another command

I would like to make it so that you can use `WithSuperUser(username, secret)` and `WithApplyMigrations()` so that people don't have to fiddle with the commands.

I think I can do this by using a `DjangoProjectResource` and have two extension methods to extend it. What do you think?

Reply:
I'm not sure about flask and django at this point. Lets get python + virtual environments sorted out first.

Reply:
Agreed, I'll focus on that first.